bricks/vs/src/code-editor/index.tsx

```diff
@@ -200,6 +201,12 @@ class CodeEditor extends FormItemElementBase implements CodeEditorProps {
   @property({ type: Boolean })
   accessor showExpandButton: boolean | undefined;
 
+  /**
+   * 行数配置
+   */
+  @property()
+  accessor lineNumber: monaco.editor.LineNumbersType | undefined;
```

Comment:
命名也和以前一样吧，不然到时 v2 构件切 v3 用错

```suggestion
  accessor lineNumbers: monaco.editor.LineNumbersType | undefined;
```